templates/master/00-master/alibabacloud/files/opt-libexec-openshift-alibabacloud-routes-sh.yaml

```diff
@@ -167,6 +189,7 @@ contents:
             list_lb_ips
             remove_stale
             add_rules
+            remove_processed_files
```

Comment:
The api-watcher seems very pervasive in overwriting and removing the up/down files. Does the unit trigger on overwrites even if the contents of the files have not changed? If that's the case, it seems this was designed to run periodically from the get go?

Anyway, the race is marginal but difficult to solve changing only the script. Even after this time we could miss a file event while the unit is still active.

Reply:
Looking at: https://github.com/openshift/machine-config-operator/blob/d112bae5ecc0322d62a606615dcb7fc5f5b64f11/cmd/apiserver-watcher/run.go#L243-L268
`apiserver-watcher` will only create the file when the state changes, not periodically